src/shared/auth.ts

```diff
@@ -144,26 +143,23 @@ export const OpenIdProviderDiscoveryMetadataSchema =
 /**
  * OAuth 2.1 token response
  */
-export const OAuthTokensSchema = z
-  .object({
-    access_token: z.string(),
-    id_token: z.string().optional(), // Optional for OAuth 2.1, but necessary in OpenID Connect
-    token_type: z.string(),
-    expires_in: z.number().optional(),
-    scope: z.string().optional(),
-    refresh_token: z.string().optional(),
-  })
-  .strip();
+export const OAuthTokensSchema = z.object({
```

Comment:
https://zod.dev/v4/changelog#deprecates-strict-and-passthrough
```suggestion
export const OAuthTokensSchema = z.strictObject({
```

also in 184 194 211 219